tasks/libs/common/utils.py

```diff
@@ -318,6 +330,18 @@ def get_git_pretty_ref():
     """
     Return the name of the current Git branch or the tag if in a detached state
     """
+    # https://docs.gitlab.com/ee/ci/variables/predefined_variables.html
+    if running_in_gitlab_ci():
+        return os.environ["CI_COMMIT_REF_NAME"]
+
+    # https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables
+    if running_in_github_actions():
+        return os.environ.get("GITHUB_HEAD_REF") or os.environ["GITHUB_REF"].split("/")[-1]
+
+    # https://circleci.com/docs/variables/#built-in-environment-variables
+    if running_in_circleci():
+        return os.environ.get("CIRCLE_TAG") or os.environ["CIRCLE_BRANCH"]
```

Comment:
```suggestion
        return os.environ["CIRCLE_BRANCH"]
```
I'm not sure we are [running for tags](https://circleci.com/docs/workflows/#executing-workflows-for-a-git-tag) on circleci, only on [branches](https://github.com/DataDog/datadog-agent/blob/main/.circleci/config.yml#L269). But this is probably harmless